src/components/UpdateVote.jsx

```diff
@@ -0,0 +1,103 @@
+import axios from "axios";
+import { useState } from "react";
+
+function UpdateVote({ vote, setVote, setArticle, article }) {
+  const [liked, setLiked] = useState(false);
+  const [unLiked, setUnLiked] = useState(false);
+  const [likeDisable, likeSetDisable] = useState(false);
+  const [unLikeDisable, unLikeSetDisable] = useState(false);
```

Comment:
these booleans seem to be changing at the same time to the same value - how is `liked` and `likeDisable` different?